js/app.js

```diff
@@ -75,40 +111,118 @@ let projects=[
           <li> <a href="#">${tech[3]}</a></li>
       </ul>
-   </div>
-   <div class="green-btn">
-        <a href="" class="viewProject">See Project</a>
+      <div class="green-btn">
+        <a href="" data-modal-target="#project-modal" class="viewProject">See Project</a>
     </div>
-    </div>`;
+   </div>
+</div>`;
+
 }
 
-document.querySelector('.projects').innerHTML=div;
+document.querySelector('.projects').innerHTML=project;
 
-const projectDetailsModal=document.querySelector('.project-details');
+const projectDetailsModal=document.querySelector('.modal');
 
-document.querySelectorAll('.viewProject').forEach((button)=>{
+document.querySelectorAll('.viewProject').forEach((button, key)=>{
     button.addEventListener('click', (e)=>{
         e.preventDefault();
-        projectDetailsModal.style.display='flex';
+        document.querySelector('.modal-overlay').classList.add('active');
+        projectDetailsModal.classList.toggle('active');
+
+        document.querySelector('.modal-overlay').addEventListener('click', ()=>{
+            document.querySelector('.modal-overlay').classList.remove('active');
+            projectDetailsModal.classList.remove('active');
+        });
+
+        let proj=projects[key];
+
+        div=`<div class="modal-header">
+             <span class="close">&times;</span>
+            </div>
+            <div class="modal-body">
+             <div class="project-image">
+               <img class="featured-image" src=${proj.featuredImage} alt="project-image" />
+             </div>
+             <div class="project-title">
+               <h2>${proj.title}</h2>
+               <ul class="view-btn view-1">
+                 <li>
+                   <a class="see-live" href=${proj.sourceURL}>See Live
+                   <img src="./assets/icons/see-icon.svg" alt="explore">
+                   </a>
+                 </li>
+                 <li>
+                   <a class="see-progress" href=${proj.versionURL}>See Progress
+                     <img src="./assets/icons/github-white.svg" alt="github">
+                   </a>
+                   
+                 </li>
+               </ul>
+             </div>
+             <ul class="frameworks">
+             <li>${proj.teachnologies[0]}</li>
+             <li>${proj.teachnologies[1]}</li>
+             <li>${proj.teachnologies[2]}</li>
+             <li>${proj.teachnologies[3]}</li>
+             </ul>
+             <div class="project-description">
+               <p>${proj.description}</p>
+             </div>
+             <ul class="view-btn view-2">
+             <li>
+               <a class="see-live" href=${proj.sourceURL}>See Live
+               <img src="./assets/icons/see-icon.svg" alt="explore">
+               </a>
+             </li>
+             <li>
+               <a class="see-progress" href=${proj.versionURL}>See Progress
+                 <img src="./assets/icons/github-white.svg" alt="github">
```

Comment:
- [ ] the see live and See Source buttons should be at the bottom of the pop up like in the Figma design please try to fix that 
- Figma 
![image](https://user-images.githubusercontent.com/61790587/169577770-5f036c4e-9776-4f9a-b665-539b1bdf63c6.png)
- Yours 
![image](https://user-images.githubusercontent.com/61790587/169577834-dab3cf0e-c4b2-4be1-a504-8a5eb6fac6dc.png)